examples/federated-learning-mistnet-yolo-aggregator.Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM tensorflow/tensorflow:1.15.4
+
```

Comment:
Seems to be more appropriate to use a `PyTorch ` or `python-slim`.

Reply:
Start from ```Python-slim``` needs to install extra libs, which may lead to larger image size.